bigchaindb/commands/bigchain.py

```diff
@@ -222,23 +243,25 @@ def run_load(args):
 
 
 def run_set_shards(args):
+    conn = backend.connect()
     for table in ['bigchain', 'backlog', 'votes']:
         # See https://www.rethinkdb.com/api/python/config/
-        table_config = r.table(table).config().run(db.get_conn())
+        table_config = conn.run(r.table(table).config())
```

Comment:
we can move this to a different module, but for now it's fine IMO (it's more important to have others working on the MongoDB queries).